test/unit/append-styles.test.js

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable no-unused-expressions */
 
 import { fixture, fixtureSync, defineCE, assert, expect, html, nextFrame } from '@open-wc/testing';
-import { TemplateElement } from 'src/TemplateElement';
```

Comment:
I would say that we should also test against our TemplateElement. But also the same tests should be run against the LitTemplateElement.